File: rename files.py

"""
Renaming files
"""

import os

def main():
    # to rename a file
    folder = "F:\\server\\htdocs\\Ibrat\\"
    if(os.path.isdir(folder)):
        oldName = folder + 'newfile.txt'
        newName = folder + 'anyrandomname.txt'
        os.rename(oldName, newName)

    # to rename multiple files with similar name
    folder = 'F:\\server\\htdocs\\Ibrat\\Experiment Folder\\'
    if(os.path.isdir(folder)):
        listName = os.listdir(folder)
        if(listName):
            for i in range (0,len(listName)):
                oldName = folder + listName[i]
                newName = folder + 'anyNewName' + str(i) + '.txt'
                os.rename(oldName, newName)

if __name__ == '__main__':
    main()

File: rename files.php
<?php

//to rename one file in a directory
$directory = '../Ibrat/';	 // folder or directory name
if (is_dir($directory)) {
	$oldFile = $directory . 'file1.txt';
	$newFile = $directory . 'newfile.txt';
	rename($oldFile, $newFile);
}

//to rename all files with a similar but not exact same name, in a directory
$directory = '../Ibrat/Experiment Folder/';	// folder or directory name
if (is_dir($directory)) {
	$filePointer = scandir($directory, 0);
	if ($filePointer) {
		for ($i=2; $i < count($filePointer); $i++) { 
			// code...
			$oldFile = $directory . $filePointer[$i];
			$newFile = $directory . 'newfile' . strval($i). '.txt';
			rename($oldFile, $newFile);
		}
	}
}

?>